pages/advanced/dvb-font-downloading.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: DVB Font Downloading
+parent: Advanced Features
+---
+
+<details  markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+# DVB Font Downloading
+
+Dash.js supports the mechanism described in the DVB DASH profile (ETSI TS 103 285 Section 7.2 Downloadable Fonts) for signalling downloadable fonts using descriptors within an MPD.
+This is intended for use with EBU-TT-D (compatible with IMSC1 text) subtitles.
+The key details of the mechanism and how to use it are covered here.
+
+## Usage
+
+As a content provider, you may choose to specify a font within your TTML subtitles that the subtitles should be rendered with.
+This could be for accessibility, language, or stylistic reasons.
+However this will only work if the specified font is available on the device or browser where the dash player is being used, which may not always be under the control of the content provider.
```

Comment:
```suggestion
However this will only work if the specified fonts are available on the device or browser where the dash player is being used, which may not always be under the control of the content provider.
```